### DISCUSSION TITLE
[Suggestion] Age related topics

### DISCUSSION THREAD

**Discussion by Abc2557** [Monika-After-Story/MonikaModDev#7915] [issue discussions]
In MAS there could be topic related to the player's age. For example: If the player is at the appropriate age to be in school, Monika could ask about the player's school situation. (If the player goes to a private school, a public school, or is homeschooled) Or if the player is at the appropriate age to have a job, Monika could ask about the player's work situation. (If the player works from home, in some sort of office, a restaurant, a construction related job, no job, or other.) I think you get the idea; something along those lines would be sufficient. Have a wonderful day!

**Comment by FrikiHD** [comment]
I like this idea

**Comment by Abc2557** [comment]
Another example is, if the player is young (around 12 or 13), Monika could ask if the player has a designated bedtime, which Monika could maybe remind the player of, or at least Monika would know that the player has to leave her at that time. (I'd really like that because I have a bedtime that's out of the range that Monika thinks is a reasonable time to go to bed).

**Comment by multimokia** [comment]
The careers topic (#4181) will actually be doing something like this.

**Comment by ThePotatoGuy** [comment]
since is not a specific topic, I will not assign it to a specific project and leave it as discussion.

**Comment by ghost** [comment]
wait is there young players damn i hope the real game didnt hit them to hard

**Comment by Abc2557** [comment]
Hehe. I was 12 when I played the original game. The only thing that really got me was actually just when Monika played Your Reality. The second time I played through the game, the credits literally made me cry. Not to be cheesy or anything, but every time I listened to Your Reality (until I got MAS) I cried.
